### ISSUE TITLE
TypeError: Cannot read property 'IS_COVERAGE_ACTIVE' of undefined

### ISSUE THREAD

**Issue by noahsw** [serut/meteor-coverage#11]
## Expected Behavior

lmieulet:meteor-coverage boots without dependency injection issue

## Current Behavior

```
╰─± meteor npm run start-coverage

> league-wins-pool@1.0.0 start-coverage /Users/noah/Projects/league-wins-pool/app
> COVERAGE=1 COVERAGE_VERBOSE=1 COVERAGE_APP_FOLDER=${PWD/#$HOME/~}/ meteor

[[[[[ ~/Projects/league-wins-pool/app ]]]]]

=> Started proxy.
=> Meteor 1.4.0.1 is available. Update this project with 'meteor update'.
=> Started MongoDB.
I20160806-23:01:19.160(-7)? Coverage active
I20160806-23:01:19.276(-7)? Reading custom configuration
I20160806-23:01:19.276(-7)? Loading default configuration: clientside.public
I20160806-23:01:19.276(-7)? Loading default configuration: serverside
I20160806-23:01:19.277(-7)? Loading default configuration: others
W20160806-23:01:19.278(-7)? (STDERR)
W20160806-23:01:19.278(-7)? (STDERR) /Users/noah/.meteor/packages/meteor-tool/.1.3.5_1.mj1j7l++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/server-lib/node_modules/fibers/future.js:280
W20160806-23:01:19.278(-7)? (STDERR)            				throw(ex);
W20160806-23:01:19.278(-7)? (STDERR)            				      ^
W20160806-23:01:19.279(-7)? (STDERR) TypeError: Cannot read property 'IS_COVERAGE_ACTIVE' of undefined
W20160806-23:01:19.279(-7)? (STDERR)     at meteorInstall.node_modules.meteor.lmieulet:meteor-coverage.server.services.instrumenter.js (packages/lmieulet:meteor-coverage/server/services/instrumenter.js:6:5)
W20160806-23:01:19.279(-7)? (STDERR)     at fileEvaluate (packages/modules-runtime/.npm/package/node_modules/install/install.js:153:1)
W20160806-23:01:19.280(-7)? (STDERR)     at require (packages/modules-runtime/.npm/package/node_modules/install/install.js:82:1)
W20160806-23:01:19.280(-7)? (STDERR)     at meteorInstall.node_modules.meteor.lmieulet:meteor-coverage.server.main.js (packages/lmieulet:meteor-coverage/server/main.js:1:1)
W20160806-23:01:19.280(-7)? (STDERR)     at fileEvaluate (packages/modules-runtime/.npm/package/node_modules/install/install.js:153:1)
W20160806-23:01:19.280(-7)? (STDERR)     at require (packages/modules-runtime/.npm/package/node_modules/install/install.js:82:1)
W20160806-23:01:19.280(-7)? (STDERR)     at node_modules/meteor/lmieulet:meteor-coverage/node_modules/mkdirp/index.js:98:1
W20160806-23:01:19.280(-7)? (STDERR)     at node_modules/meteor/lmieulet:meteor-coverage/node_modules/mkdirp/index.js:98:1
W20160806-23:01:19.281(-7)? (STDERR)     at /Users/noah/Projects/league-wins-pool/app/.meteor/local/build/programs/server/boot.js:297:10
W20160806-23:01:19.281(-7)? (STDERR)     at Array.forEach (native)
=> Exited with code: 8
```

## Steps to Reproduce

Update previously working build to lmieulet:meteor-coverage@0.9.0

**Comment by serut** [comment]
I already saw that issue, thank you for reporting it.
Did you clone meteor-coverage inside the packages folder ? It could be the source of the issue

And I rewrite your issue to remove useless sections

**Comment by j6k4m8** [comment]
Also experiencing this, but can confirm that I simply installed by adding to `packages`; no clone here.

**Comment by serut** [comment]
Thanks @j6k4m8. I review the code of this package and I still fail to understand the phenomenon...
Can you confirm that you do not instrument meteor-coverage. I don't know if you add the .coverage.json file in the root folder of you app, but if you can run the test with COVERAGE_VERBOSE=1 to see if the server side of this library have been instrumented and if it has an impact.

**Comment by j6k4m8** [comment]
Even when I run with `COVERAGE_VERBOSE=1`, I still get the same output:

```
I20160815-10:28:13.408(-4)? Coverage active
I20160815-10:28:13.437(-4)? Loading default configuration: clientside.*
I20160815-10:28:13.437(-4)? Loading default configuration: serverside
I20160815-10:28:13.438(-4)? Loading default configuration: others
I20160815-10:28:13.438(-4)? Loading default configuration: others
W20160815-10:28:13.439(-4)? (STDERR)
W20160815-10:28:13.439(-4)? (STDERR) /Users/jordan/.meteor/packages/meteor-tool/.1.3.5_1.gdx122++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/server-lib/node_modules/fibers/future.js:280
W20160815-10:28:13.440(-4)? (STDERR)            				throw(ex);
W20160815-10:28:13.440(-4)? (STDERR)            				      ^
W20160815-10:28:13.538(-4)? (STDERR) TypeError: Cannot read property 'IS_COVERAGE_ACTIVE' of undefined
W20160815-10:28:13.539(-4)? (STDERR)     at meteorInstall.node_modules.meteor.lmieulet:meteor-coverage.server.services.instrumenter.js (packages/lmieulet:meteor-coverage/server/services/instrumenter.js:6:5)
W20160815-10:28:13.539(-4)? (STDERR)     at fileEvaluate (packages/modules-runtime/.npm/package/node_modules/install/install.js:153:1)
W20160815-10:28:13.539(-4)? (STDERR)     at require (packages/modules-runtime/.npm/package/node_modules/install/install.js:82:1)
W20160815-10:28:13.539(-4)? (STDERR)     at meteorInstall.node_modules.meteor.lmieulet:meteor-coverage.server.main.js (packages/lmieulet:meteor-coverage/server/main.js:1:1)
W20160815-10:28:13.540(-4)? (STDERR)     at fileEvaluate (packages/modules-runtime/.npm/package/node_modules/install/install.js:153:1)
W20160815-10:28:13.540(-4)? (STDERR)     at require (packages/modules-runtime/.npm/package/node_modules/install/install.js:82:1)
W20160815-10:28:13.540(-4)? (STDERR)     at node_modules/meteor/lmieulet:meteor-coverage/node_modules/mkdirp/index.js:98:1
W20160815-10:28:13.540(-4)? (STDERR)     at node_modules/meteor/lmieulet:meteor-coverage/node_modules/mkdirp/index.js:98:1
W20160815-10:28:13.540(-4)? (STDERR)     at /my/app/path/.meteor/local/build/programs/server/boot.js:297:10
W20160815-10:28:13.541(-4)? (STDERR)     at Array.forEach (native)
W20160815-10:28:13.541(-4)? (STDERR)     at Function._.each._.forEach (/Users/jordan/.meteor/packages/meteor-tool/.1.3.5_1.gdx122++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/server-lib/node_modules/underscore/underscore.js:79:11)
W20160815-10:28:13.542(-4)? (STDERR)     at /my/app/path/.meteor/local/build/programs/server/boot.js:133:5
```

**Comment by j6k4m8** [comment]
I don't use a `.coverage.json` file, if that's useful information.

**Comment by serut** [comment]
Can you try again with the last update `lmieulet:meteor-coverage@0.9.4` ?

**Comment by spectraflex** [comment]
+1 
Same issue on windows with Version 0.9.4.

**Comment by serut** [comment]
Ok. Can you provide a reproducible environment or some log ? 
Because when I test more than [150 times meteor-coverage-app-exemple](https://circleci.com/gh/serut/meteor-coverage-app-exemple/73) I don't have this issue.

**Comment by spectraflex** [comment]
Ok, here are the logs, by the way with 0.8.1 everything works fine.

`W20160901-09:38:23.626(2)? (STDERR) \AppData\Local\.meteor\packages\meteor-tool\1.3.3_1\mt-os.windows.x86_32\dev_bundle\server-lib\node_modules\fibers\future.js:280
W20160901-09:38:23.626(2)? (STDERR) 						throw(ex);
W20160901-09:38:23.626(2)? (STDERR) 						      ^
W20160901-09:38:23.626(2)? (STDERR) TypeError: Cannot read property 'IS_COVERAGE_ACTIVE' of undefined
W20160901-09:38:23.626(2)? (STDERR)     at meteorInstall.node_modules.meteor.lmieulet:meteor-coverage.server.index.js (meteor://ðŸ’»app/packages/lmieulet:meteor-coverage/server/index.js:10:5)
W20160901-09:38:23.641(2)? (STDERR)     at fileEvaluate (meteor://ðŸ’»app/packages/modules-runtime/.npm/package/node_modules/install/install.js:153:1)
W20160901-09:38:23.641(2)? (STDERR)     at require (meteor://ðŸ’»app/packages/modules-runtime/.npm/package/node_modules/install/install.js:82:1)
W20160901-09:38:23.641(2)? (STDERR)     at meteor://ðŸ’»app/node_modules/meteor/lmieulet:meteor-coverage/node_modules/qs/lib/parse.js:167:1
W20160901-09:38:23.641(2)? (STDERR)     at meteor://ðŸ’»app/node_modules/meteor/lmieulet:meteor-coverage/node_modules/qs/lib/parse.js:167:1
W20160901-09:38:23.641(2)? (STDERR)     at .meteor\local\build\programs\server\boot.js:297:10
W20160901-09:38:23.641(2)? (STDERR)     at Function._.each._.forEach (node_modules\underscore\underscore.js:153:9)
W20160901-09:38:23.641(2)? (STDERR)     at .meteor\local\build\programs\server\boot.js:133:5`

**Comment by serut** [comment]
Sorry if it bothers you, but can you retry a last time with `COVERAGE_VERBOSE=1`. You will see there is more log
Thanks in advance

**Comment by spectraflex** [comment]
Ok here are the log with COVERAGE_VERBOSE=1. Same log in this case?

```
W20160901-10:06:26.814(2)? (STDERR) \AppData\Local\.meteor\packages\meteor-tool\1.3.3_1\mt-os.windows.x86_32\dev_bundle\server-lib\node_modules\fibers\future.js:280
W20160901-10:06:26.815(2)? (STDERR) 						throw(ex);
W20160901-10:06:26.815(2)? (STDERR) 						      ^
W20160901-10:06:26.815(2)? (STDERR) TypeError: Cannot read property 'IS_COVERAGE_ACTIVE' of undefined
W20160901-10:06:26.815(2)? (STDERR)     at meteorInstall.node_modules.meteor.lmieulet:meteor-coverage.server.index.js (meteor://ðŸ’»app/packages/lmieulet:meteor-coverage/server/index.js:10:5)
W20160901-10:06:26.816(2)? (STDERR)     at fileEvaluate (meteor://ðŸ’»app/packages/modules-runtime/.npm/package/node_modules/install/install.js:153:1)
W20160901-10:06:26.816(2)? (STDERR)     at require (meteor://ðŸ’»app/packages/modules-runtime/.npm/package/node_modules/install/install.js:82:1)
W20160901-10:06:26.816(2)? (STDERR)     at meteor://ðŸ’»app/node_modules/meteor/lmieulet:meteor-coverage/node_modules/qs/lib/parse.js:167:1
W20160901-10:06:26.816(2)? (STDERR)     at meteor://ðŸ’»app/node_modules/meteor/lmieulet:meteor-coverage/node_modules/qs/lib/parse.js:167:1
W20160901-10:06:26.817(2)? (STDERR)     at .meteor\local\build\programs\server\boot.js:297:10
W20160901-10:06:26.817(2)? (STDERR)     at Function._.each._.forEach (node_modules\underscore\underscore.js:153:9)
W20160901-10:06:26.817(2)? (STDERR)     at .meteor\local\build\programs\server\boot.js:133:5
```

**Comment by serut** [comment]
Nop, it's not normal, you should see the log from https://github.com/serut/meteor-coverage/blob/master/server/context/conf.js#L86
Can you clone meteor-coverage inside your app and retest ?
```
cd packages && git clone https://github.com/serut/meteor-coverage
SET COVERAGE 1
SET COVERAGE_VERBOSE 1
SET # it prints all env variables, you can check if it's ok
meteor
```

Then if it's still the same issue, can you add : 
- inside `meteor-coverage/server/index.js`
`console.log(Conf);` on line 3

- inside `/server/context/conf.js`
`console.log(COVERAGE_EXPORT_FOLDER, exclude, include, Log)` on line 93

- inside `/server/context/log.js`
`console.log(COVERAGE_VERBOSE);` on line 2

**Comment by serut** [comment]
@noahsw 
Is this repository https://github.com/winsleague/winsleague a reproducible environment ?

**Comment by noahsw** [comment]
Yep.

I cloned meteor-coverage and added your debugging comments, with the name of the file in the comment so it's easier to see where it's coming from.

It looks like the culprit is `console.log(Conf)`. That's printing undefined.

This was the output:
```
$ COVERAGE=1 COVERAGE_VERBOSE=1 COVERAGE_APP_FOLDER=${PWD/#$HOME/~}/ meteor

=> Started proxy.
=> Meteor 1.4.1.1 is available. Update this project with 'meteor update'.
=> Started MongoDB.
I20160905-21:51:53.155(-7)? log.js true
I20160905-21:51:53.156(-7)? Coverage active
I20160905-21:51:53.157(-7)? Loading default configuration
I20160905-21:51:53.157(-7)? Coverage configuration:
I20160905-21:51:53.157(-7)? - IS_COVERAGE_ACTIVE= true
I20160905-21:51:53.157(-7)? - COVERAGE_APP_FOLDER= /Users/noah/Projects/league-wins-pool/app/
I20160905-21:51:53.158(-7)? .coverage.json values:
I20160905-21:51:53.158(-7)? - exclude= { general: [ '/**/*test.js', '/**/lib/test/**/*', '/**/tests/**/*' ],
I20160905-21:51:53.159(-7)?   server:
I20160905-21:51:53.159(-7)?    [ '/**/lib/test/**/*',
I20160905-21:51:53.159(-7)?      '/**/server/test/**/*',
I20160905-21:51:53.160(-7)?      '**/tests/**/*',
I20160905-21:51:53.160(-7)?      '/packages/global-imports.js' ],
I20160905-21:51:53.160(-7)?   client:
I20160905-21:51:53.160(-7)?    [ '/**/client/test/**/*.js',
I20160905-21:51:53.161(-7)?      '/**/client/tests/**/*.js',
I20160905-21:51:53.161(-7)?      '/**/client/test/**/*.js',
I20160905-21:51:53.161(-7)?      '/**/client/stylesheets/**/*',
I20160905-21:51:53.161(-7)?      '**/underscore.js',
I20160905-21:51:53.161(-7)?      '**/meteor.js',
I20160905-21:51:53.161(-7)?      '**/meteor-base.js',
I20160905-21:51:53.162(-7)?      '**/mobile-experience.js',
I20160905-21:51:53.162(-7)?      '**/babel-compiler.js',
I20160905-21:51:53.162(-7)?      '**/ecmascript.js',
I20160905-21:51:53.162(-7)?      '**/base64.js',
I20160905-21:51:53.163(-7)?      '**/ejson.js',
I20160905-21:51:53.163(-7)?      '**/id-map.js',
I20160905-21:51:53.163(-7)?      '**/ordered-dict.js',
I20160905-21:51:53.163(-7)?      '**/tracker.js',
I20160905-21:51:53.163(-7)?      '**/modules-runtime.js',
I20160905-21:51:53.164(-7)?      '**/modules.js',
I20160905-21:51:53.164(-7)?      '**/es5-shim.js',
I20160905-21:51:53.164(-7)?      '**/promise.js',
I20160905-21:51:53.164(-7)?      '**/ecmascript-runtime.js',
I20160905-21:51:53.165(-7)?      '**/babel-runtime.js',
I20160905-21:51:53.165(-7)?      '**/random.js',
I20160905-21:51:53.165(-7)?      '**/mongo-id.js',
I20160905-21:51:53.165(-7)?      '**/diff-sequence.js',
I20160905-21:51:53.165(-7)?      '**/geojson-utils.js',
I20160905-21:51:53.166(-7)?      '**/minimongo.js',
I20160905-21:51:53.166(-7)?      '**/check.js',
I20160905-21:51:53.166(-7)?      '**/retry.js',
I20160905-21:51:53.167(-7)?      '**/ddp-common.js',
I20160905-21:51:53.167(-7)?      '**/reload.js',
I20160905-21:51:53.168(-7)?      '**/ddp-client.js',
I20160905-21:51:53.168(-7)?      '**/ddp.js',
I20160905-21:51:53.169(-7)?      '**/ddp-server.js',
I20160905-21:51:53.169(-7)?      '**/allow-deny.js',
I20160905-21:51:53.169(-7)?      '**/insecure.js',
I20160905-21:51:53.169(-7)?      '**/mongo.js',
I20160905-21:51:53.169(-7)?      '**/blaze-html-templates.js',
I20160905-21:51:53.169(-7)?      '**/reactive-var.js',
I20160905-21:51:53.170(-7)?      '**/jquery.js',
I20160905-21:51:53.170(-7)?      '**/standard-minifier-css.js',
I20160905-21:51:53.170(-7)?      '**/standard-minifier-js.js',
I20160905-21:51:53.170(-7)?      '**/autopublish.js',
I20160905-21:51:53.171(-7)?      '**/dburles_eslint.js',
I20160905-21:51:53.171(-7)?      '**/webapp.js',
I20160905-21:51:53.171(-7)?      '**/livedata.js',
I20160905-21:51:53.171(-7)?      '**/hot-code-push.js',
I20160905-21:51:53.171(-7)?      '**/deps.js',
I20160905-21:51:53.172(-7)?      '**/htmljs.js',
I20160905-21:51:53.172(-7)?      '**/observe-sequence.js',
I20160905-21:51:53.172(-7)?      '**/blaze.js',
I20160905-21:51:53.172(-7)?      '**/spacebars.js',
I20160905-21:51:53.172(-7)?      '**/templating.js',
I20160905-21:51:53.173(-7)?      '**/launch-screen.js',
I20160905-21:51:53.173(-7)?      '**/ui.js',
I20160905-21:51:53.173(-7)?      '**/autoupdate.js',
I20160905-21:51:53.173(-7)?      '**/global-imports.js',
I20160905-21:51:53.174(-7)?      '**/practicalmeteor_mocha.js',
I20160905-21:51:53.174(-7)?      '**/practicalmeteor_loglevel.js',
I20160905-21:51:53.174(-7)?      '**/practicalmeteor_sinon.js',
I20160905-21:51:53.174(-7)?      '**/practicalmeteor_mocha-console-runner.js',
I20160905-21:51:53.174(-7)?      '**/practicalmeteor_mocha-core.js',
I20160905-21:51:53.175(-7)?      '**/kadira_flow-router.js',
I20160905-21:51:53.175(-7)?      '**/tmeasday_test-reporter-helpers.js',
I20160905-21:51:53.175(-7)?      '**/practicalmeteor_chai.js' ] }
I20160905-21:51:53.175(-7)? - include= [ '**/lib/**/methods.js' ]
I20160905-21:51:53.176(-7)? - COVERAGE_EXPORT_FOLDER= ./.coverage
I20160905-21:51:53.206(-7)? conf.js ./.coverage { general: [ '/**/*test.js', '/**/lib/test/**/*', '/**/tests/**/*' ],
I20160905-21:51:53.206(-7)?   server:
I20160905-21:51:53.206(-7)?    [ '/**/lib/test/**/*',
I20160905-21:51:53.206(-7)?      '/**/server/test/**/*',
I20160905-21:51:53.206(-7)?      '**/tests/**/*',
I20160905-21:51:53.207(-7)?      '/packages/global-imports.js' ],
I20160905-21:51:53.207(-7)?   client:
I20160905-21:51:53.207(-7)?    [ '/**/client/test/**/*.js',
I20160905-21:51:53.207(-7)?      '/**/client/tests/**/*.js',
I20160905-21:51:53.208(-7)?      '/**/client/test/**/*.js',
I20160905-21:51:53.208(-7)?      '/**/client/stylesheets/**/*',
I20160905-21:51:53.208(-7)?      '**/underscore.js',
I20160905-21:51:53.208(-7)?      '**/meteor.js',
I20160905-21:51:53.211(-7)?      '**/meteor-base.js',
I20160905-21:51:53.211(-7)?      '**/mobile-experience.js',
I20160905-21:51:53.211(-7)?      '**/babel-compiler.js',
I20160905-21:51:53.211(-7)?      '**/ecmascript.js',
I20160905-21:51:53.211(-7)?      '**/base64.js',
I20160905-21:51:53.212(-7)?      '**/ejson.js',
I20160905-21:51:53.212(-7)?      '**/id-map.js',
I20160905-21:51:53.212(-7)?      '**/ordered-dict.js',
I20160905-21:51:53.212(-7)?      '**/tracker.js',
I20160905-21:51:53.213(-7)?      '**/modules-runtime.js',
I20160905-21:51:53.213(-7)?      '**/modules.js',
I20160905-21:51:53.213(-7)?      '**/es5-shim.js',
I20160905-21:51:53.213(-7)?      '**/promise.js',
I20160905-21:51:53.214(-7)?      '**/ecmascript-runtime.js',
I20160905-21:51:53.214(-7)?      '**/babel-runtime.js',
I20160905-21:51:53.214(-7)?      '**/random.js',
I20160905-21:51:53.214(-7)?      '**/mongo-id.js',
I20160905-21:51:53.214(-7)?      '**/diff-sequence.js',
I20160905-21:51:53.215(-7)?      '**/geojson-utils.js',
I20160905-21:51:53.215(-7)?      '**/minimongo.js',
I20160905-21:51:53.216(-7)?      '**/check.js',
I20160905-21:51:53.216(-7)?      '**/retry.js',
I20160905-21:51:53.217(-7)?      '**/ddp-common.js',
I20160905-21:51:53.217(-7)?      '**/reload.js',
I20160905-21:51:53.217(-7)?      '**/ddp-client.js',
I20160905-21:51:53.217(-7)?      '**/ddp.js',
I20160905-21:51:53.217(-7)?      '**/ddp-server.js',
I20160905-21:51:53.218(-7)?      '**/allow-deny.js',
I20160905-21:51:53.218(-7)?      '**/insecure.js',
I20160905-21:51:53.218(-7)?      '**/mongo.js',
I20160905-21:51:53.218(-7)?      '**/blaze-html-templates.js',
I20160905-21:51:53.218(-7)?      '**/reactive-var.js',
I20160905-21:51:53.218(-7)?      '**/jquery.js',
I20160905-21:51:53.219(-7)?      '**/standard-minifier-css.js',
I20160905-21:51:53.219(-7)?      '**/standard-minifier-js.js',
I20160905-21:51:53.219(-7)?      '**/autopublish.js',
I20160905-21:51:53.219(-7)?      '**/dburles_eslint.js',
I20160905-21:51:53.219(-7)?      '**/webapp.js',
I20160905-21:51:53.220(-7)?      '**/livedata.js',
I20160905-21:51:53.221(-7)?      '**/hot-code-push.js',
I20160905-21:51:53.221(-7)?      '**/deps.js',
I20160905-21:51:53.221(-7)?      '**/htmljs.js',
I20160905-21:51:53.221(-7)?      '**/observe-sequence.js',
I20160905-21:51:53.221(-7)?      '**/blaze.js',
I20160905-21:51:53.222(-7)?      '**/spacebars.js',
I20160905-21:51:53.222(-7)?      '**/templating.js',
I20160905-21:51:53.222(-7)?      '**/launch-screen.js',
I20160905-21:51:53.222(-7)?      '**/ui.js',
I20160905-21:51:53.222(-7)?      '**/autoupdate.js',
I20160905-21:51:53.222(-7)?      '**/global-imports.js',
I20160905-21:51:53.223(-7)?      '**/practicalmeteor_mocha.js',
I20160905-21:51:53.223(-7)?      '**/practicalmeteor_loglevel.js',
I20160905-21:51:53.223(-7)?      '**/practicalmeteor_sinon.js',
I20160905-21:51:53.223(-7)?      '**/practicalmeteor_mocha-console-runner.js',
I20160905-21:51:53.223(-7)?      '**/practicalmeteor_mocha-core.js',
I20160905-21:51:53.223(-7)?      '**/kadira_flow-router.js',
I20160905-21:51:53.225(-7)?      '**/tmeasday_test-reporter-helpers.js',
I20160905-21:51:53.225(-7)?      '**/practicalmeteor_chai.js' ] } [ '**/lib/**/methods.js' ] { error: [Function: error],
I20160905-21:51:53.225(-7)?   info: [Function: info],
I20160905-21:51:53.225(-7)?   time: [Function: time],
I20160905-21:51:53.225(-7)?   timeEnd: [Function: timeEnd] }
I20160905-21:51:53.225(-7)? index.js undefined
W20160905-21:51:53.240(-7)? (STDERR)
W20160905-21:51:53.242(-7)? (STDERR) /Users/noah/.meteor/packages/meteor-tool/.1.3.5_1.mj1j7l++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/server-lib/node_modules/fibers/future.js:280
W20160905-21:51:53.242(-7)? (STDERR)                   				throw(ex);
W20160905-21:51:53.243(-7)? (STDERR)                   				      ^
W20160905-21:51:53.416(-7)? (STDERR) TypeError: Cannot read property 'IS_COVERAGE_ACTIVE' of undefined
W20160905-21:51:53.416(-7)? (STDERR)     at meteorInstall.node_modules.meteor.lmieulet:meteor-coverage.server.index.js (packages/lmieulet:meteor-coverage/server/index.js:12:5)
W20160905-21:51:53.416(-7)? (STDERR)     at fileEvaluate (packages/modules-runtime/.npm/package/node_modules/install/install.js:153:1)
W20160905-21:51:53.417(-7)? (STDERR)     at require (packages/modules-runtime/.npm/package/node_modules/install/install.js:82:1)
W20160905-21:51:53.417(-7)? (STDERR)     at node_modules/meteor/lmieulet:meteor-coverage/node_modules/qs/lib/parse.js:167:1
W20160905-21:51:53.417(-7)? (STDERR)     at node_modules/meteor/lmieulet:meteor-coverage/node_modules/qs/lib/parse.js:167:1
W20160905-21:51:53.417(-7)? (STDERR)     at /Users/noah/Projects/league-wins-pool/app/.meteor/local/build/programs/server/boot.js:297:10
W20160905-21:51:53.417(-7)? (STDERR)     at Array.forEach (native)
W20160905-21:51:53.417(-7)? (STDERR)     at Function._.each._.forEach (/Users/noah/.meteor/packages/meteor-tool/.1.3.5_1.mj1j7l++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/server-lib/node_modules/underscore/underscore.js:79:11)
W20160905-21:51:53.418(-7)? (STDERR)     at /Users/noah/Projects/league-wins-pool/app/.meteor/local/build/programs/server/boot.js:133:5
```

**Comment by noahsw** [comment]
I seemed to resolve the undefined error by refactoring `conf.js`. I used this style of export instead and the references now work:

```
export default {
  IS_COVERAGE_ACTIVE: _IS_COVERAGE_ACTIVE,

  COVERAGE_APP_FOLDER: _COVERAGE_APP_FOLDER,

  COVERAGE_EXPORT_FOLDER: _COVERAGE_EXPORT_FOLDER,

  exclude: _exclude,

  include: _include,
}
```

I'm still not up and running but I'll report that separately...

**Comment by serut** [comment]
Fixed by meteor-coverage `v1.1.3`. Since the `v~0.9`, we have some ES6 code and node code that uses node `>v0.11`.
Meteor apps on version `1.3` uses `node 0.10`, so it can just fail.
